tests/acceptance/helpers/httpHelper.js

```diff
@@ -49,3 +63,261 @@ exports.checkOCSStatus = function (response, message) {
     throw Error(message + ' Status:' + statusCode)
   }
 }
+
+/**
+ *
+ * @param {object} payload
+ * @param {string} userId
+ * @param {object} header
+ * @param {object} urlData
+ *
+ * @returns {node-fetch}
+ */
+exports.requestEndpoint = function (urlData, payload, userId = 'admin', header = {}) {
+  const { urlParams, backendUrlString, app, appParams = '', ocsDirectoryPath = '' } = urlData
+  let davPath
+  if (urlParams) {
+    const [user, element] = urlParams
+    davPath = createDavPath(user, element)
+  } else if (backendUrlString) {
+    const backendUrl = getBackEndUrl(backendUrlString)
+    const appPath = getAppPath(app)
+    davPath = join(backendUrl, ocsDirectoryPath, appPath, appParams)
+  }
+  console.log(davPath)
```

Comment:
debug line??

Reply:
@kiranparajuli589 read the updated comment above